### PR TITLE
chore: add full CR example

### PR DESCRIPTION
**Pull request by rollandf**
- Add full NCP example
- Specify full container image in platform page
- Update versions to 24.10 beta4

### CODE REVIEW

**Comment by rollandf** [comment]
image

File: docs/customizations/cr-full-example.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  limitations under the License.
+
+.. headings # #, * *, =, -, ^, "
+.. include:: ../common/vars.rst
+
+****************************************
+NicClusterPolicy Custom Resource Example
+****************************************
+
+The following NIC Cluster Policy example contains all the sub-components that NVIDA Network Operator can deploy.
```
Comment (adrianchiris) [inline docs/customizations/cr-full-example.rst:24]: just to be extra clear, can you add something in the lines of: 

"This example should serve as reference, it is not recommended to apply it as is to your cluster."

**Comment by adrianchiris** [comment]
minor comments otherwise lgtm

File: docs/customizations/customization.rst [hunk 2]
```diff
@@ -22,4 +22,5 @@ Customization Options
 
 .. toctree::
    Helm Chart <helm.rst>
-   CRDs <crds.rst>
\ No newline at end of file
+   CRDs <crds.rst>
+   NicClusterPolicy example <cr-full-example.rst>
```
[the remark this answers is not in the document]
Comment (rollandf) [reply docs/customizations/customization.rst]: Done

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**